src/init.c

```diff
@@ -1290,7 +1306,8 @@ int main(int argc, char *argv[])
 		return -1;
 	}
 
-	if (mount("devpts", "/dev/pts", "devpts", MS_NOSUID| MS_NOEXEC, NULL) == -1) {
+	if (system_mount("devpts", "/dev/pts", "devpts", MS_NOSUID| MS_NOEXEC,
+			 NULL) == -1) {
 		perror("mount devpts failed");
 		return -1;
 	}
```

Comment:
Oh, sure, I'll have a look.

Currently it's only the system mounts that are causing problems because they are actually trying to remount the already mounted file systems, so might just pull those in a separate function.